tools/travis-install-dependencies.sh

```diff
@@ -96,3 +96,4 @@ if [ ! -f $CACHE_CCACHE_TOKEN ]; then
     # Create token
     touch $CACHE_CCACHE_TOKEN
 fi
+
```

Comment:
Nothing should be changed here

Reply:
Rolled this back in https://github.com/precice/precice/pull/421/commits/5f7809a8c733f007817750b73e41e98d4d06a016